diff -NaurwB lcdproc-0.5.5.orig/acinclude.m4 lcdproc-0.5.5/acinclude.m4
--- lcdproc-0.5.5.orig/acinclude.m4	2011-08-14 14:29:14.000000000 +0200
+++ lcdproc-0.5.5/acinclude.m4	2012-09-05 15:42:08.719066156 +0200
@@ -198,7 +198,7 @@
 		hd44780)
 			HD44780_DRIVERS="hd44780-hd44780-serial.o hd44780-hd44780-lis2.o hd44780-hd44780-usblcd.o"
 			if test "$ac_cv_port_have_lpt" = yes ; then
-				HD44780_DRIVERS="$HD44780_DRIVERS hd44780-hd44780-4bit.o hd44780-hd44780-ext8bit.o hd44780-lcd_sem.o hd44780-hd44780-winamp.o hd44780-hd44780-serialLpt.o"
+				HD44780_DRIVERS="$HD44780_DRIVERS hd44780-hd44780-4bit.o hd44780-hd44780-ext8bit.o hd44780-hd44780-aaeon.o hd44780-hd44780-advantech.o hd44780-lcd_sem.o hd44780-hd44780-winamp.o hd44780-hd44780-serialLpt.o"
 			fi
 			if test "$enable_libusb" = yes ; then
 				HD44780_DRIVERS="$HD44780_DRIVERS hd44780-hd44780-bwct-usb.o hd44780-hd44780-lcd2usb.o hd44780-hd44780-uss720.o hd44780-hd44780-usbtiny.o"
diff -NaurwB lcdproc-0.5.5.orig/configure lcdproc-0.5.5/configure
--- lcdproc-0.5.5.orig/configure	2011-11-09 20:56:35.000000000 +0100
+++ lcdproc-0.5.5/configure	2012-09-05 15:42:08.723066281 +0200
@@ -7907,7 +7907,7 @@
 		hd44780)
 			HD44780_DRIVERS="hd44780-hd44780-serial.o hd44780-hd44780-lis2.o hd44780-hd44780-usblcd.o"
 			if test "$ac_cv_port_have_lpt" = yes ; then
-				HD44780_DRIVERS="$HD44780_DRIVERS hd44780-hd44780-4bit.o hd44780-hd44780-ext8bit.o hd44780-lcd_sem.o hd44780-hd44780-winamp.o hd44780-hd44780-serialLpt.o"
+				HD44780_DRIVERS="$HD44780_DRIVERS hd44780-hd44780-4bit.o hd44780-hd44780-ext8bit.o hd44780-hd44780-aaeon.o hd44780-hd44780-advantech.o hd44780-lcd_sem.o hd44780-hd44780-winamp.o hd44780-hd44780-serialLpt.o"
 			fi
 			if test "$enable_libusb" = yes ; then
 				HD44780_DRIVERS="$HD44780_DRIVERS hd44780-hd44780-bwct-usb.o hd44780-hd44780-lcd2usb.o hd44780-hd44780-uss720.o hd44780-hd44780-usbtiny.o"
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/hd44780-aaeon.c lcdproc-0.5.5/server/drivers/hd44780-aaeon.c
--- lcdproc-0.5.5.orig/server/drivers/hd44780-aaeon.c	1970-01-01 01:00:00.000000000 +0100
+++ lcdproc-0.5.5/server/drivers/hd44780-aaeon.c	2012-09-05 15:42:20.942252980 +0200
@@ -0,0 +1,175 @@
+/** \file server/drivers/hd44780-aaeon.c
+ * \c 8bit connection type of \c hd44780 driver for Hitachi HD44780 based LCD displays.
+ *
+ * The LCD is operated in its 8 bit-mode to be connected to a single PC parallel port.
+ */
+
+/* Copyright (c)  1999, 1995 Benjamin Tse <blt@Comports.com>
+ *                2001 Joris Robijn <joris@robijn.net>
+ *                
+ * Changed connection for Aaeon FWS-7600 Network Appliance (internal LCM)
+ *                2011 Malte Pöggel <malte@maltepoeggel.de>
+ *
+ * The connections are:
+ * printer port   LCD
+ * D0  (2)	  D0 (7)
+ * D1  (3)	  D1 (8)
+ * D2  (4)	  D2 (9)
+ * D3  (5)	  D3 (10)
+ * D4  (6)	  D4 (11)
+ * D5  (7)	  D5 (12)
+ * D6  (8)	  D6 (13)
+ * D7  (9)	  D7 (14)
+ * SEL (1)        RS (4)
+ * -              RW (5) (pulled low)
+ * nLF  (16)      EN (6)
+ *
+ *
+ * Keypad connection:
+ *
+ * Key pressed: nACK (10)
+ *              PE   (12)  nFAULT (15)              
+ * Key ESC       0          0
+ * Key Down      1          1
+ * Key Up        0          1
+ * Key Enter     1          0
+ * Both bits seem to be latched (by bios?) they always stay
+ * in the state of the key pressed last.
+ *
+ *
+ * Created modular driver Dec 1999, Benjamin Tse <blt@Comports.com>
+ *
+ * Based on the code in the lcdtime package which uses the LCD
+ * controller in its 8-bit mode.
+ *
+ * This file is released under the GNU General Public License. Refer to the
+ * COPYING file distributed with this package.
+ */
+
+#include "hd44780-aaeon.h"
+#include "hd44780-low.h"
+#include "lpt-port.h"
+#include "port.h"
+#include "lcd_sem.h"
+#include "report.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+// Generally, any function that accesses the LCD control lines needs to be
+// implemented separately for each HW design. This is typically (but not
+// restricted to):
+// HD44780_senddata
+// HD44780_readkeypad
+
+void aaeon_HD44780_senddata(PrivateData *p, unsigned char displayID, unsigned char flags, unsigned char ch);
+unsigned char aaeon_HD44780_scankeypad(PrivateData *p);
+void aaeon_HD44780_output(PrivateData *p, int data);
+
+#define RS	SEL
+#define RW	0x00
+#define EN1	LF
+
+static int semid;
+
+
+/**
+ * Initialize the driver.
+ * \param drvthis  Pointer to driver structure.
+ * \retval 0       Success.
+ * \retval -1      Error.
+ */
+int
+hd_init_aaeon(Driver *drvthis)
+{
+	PrivateData *p = (PrivateData*) drvthis->private_data;
+	HD44780_functions *hd44780_functions = p->hd44780_functions;
+
+	semid = sem_get();
+
+	// Reserve the port registers
+	if (port_access_multiple(p->port,3)) {
+		report(RPT_ERR, "%s: cannot get IO-permission for 0x%03X: %s",
+				drvthis->name, p->port, strerror(errno));
+		return -1;
+	}
+
+	hd44780_functions->senddata = aaeon_HD44780_senddata;
+	hd44780_functions->scankeypad = aaeon_HD44780_scankeypad;
+	
+	// setup the lcd in 8 bit mode
+	hd44780_functions->senddata(p, 0, RS_INSTR, FUNCSET | IF_8BIT);
+	hd44780_functions->uPause(p, 4100);
+	hd44780_functions->senddata(p, 0, RS_INSTR, FUNCSET | IF_8BIT);
+	hd44780_functions->uPause(p, 100);
+	hd44780_functions->senddata(p, 0, RS_INSTR, FUNCSET | IF_8BIT | TWOLINE | SMALLCHAR);
+	hd44780_functions->uPause(p, 40);
+
+	common_init (p, IF_8BIT);
+
+	return 0;
+}
+
+
+/**
+ * Send data or commands to the display.
+ * \param p          Pointer to driver's private data structure.
+ * \param displayID  ID of the display (or 0 for all) to send data to.
+ * \param flags      Defines whether to end a command or data.
+ * \param ch         The value to send.
+ */
+void
+aaeon_HD44780_senddata(PrivateData *p, unsigned char displayID, unsigned char flags, unsigned char ch)
+{
+	unsigned char enableLines = 0, portControl;
+
+	// Only one controller is supported
+	enableLines = EN1;
+
+	if (flags == RS_INSTR)
+		portControl = 0;
+	else // RS_DATA
+		portControl = RS;
+
+	portControl |= p->backlight_bit;
+
+	sem_wait(semid);
+	port_out(p->port + 2, portControl ^ OUTMASK);
+	port_out(p->port, ch);
+	if (p->delayBus) p->hd44780_functions->uPause(p, 1);
+	port_out(p->port + 2, (enableLines|portControl) ^ OUTMASK);
+	if (p->delayBus) p->hd44780_functions->uPause(p, 1);
+	port_out(p->port + 2, portControl ^ OUTMASK);
+	sem_signal(semid);
+}
+
+
+/**
+ * Read keypress.
+ * \param p  Pointer to driver's private data structure.
+ * \return   Bitmap of the pressed keys.
+ */
+unsigned char aaeon_HD44780_scankeypad(PrivateData *p)
+{
+	unsigned char readval;
+
+	sem_wait(semid);
+
+	// Read inputs
+	readval = ~ port_in(p->port + 1) ^ INMASK;
+
+	sem_signal(semid);
+
+	readval = ~readval;	//reverse logic
+
+        // Keys down?
+        if((readval & ACK) / ACK)
+        {
+                if(((readval & PAPEREND) / PAPEREND) && ((readval & FAULT) / FAULT)) return 2;
+                        else if((readval & PAPEREND) / PAPEREND) return 4;
+                                else if((readval & nFAULT) / nFAULT) return 3;
+                                        else return 1;
+        } 
+        return 0;
+}
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/hd44780-aaeon.h lcdproc-0.5.5/server/drivers/hd44780-aaeon.h
--- lcdproc-0.5.5.orig/server/drivers/hd44780-aaeon.h	1970-01-01 01:00:00.000000000 +0100
+++ lcdproc-0.5.5/server/drivers/hd44780-aaeon.h	2012-09-05 15:42:08.724066296 +0200
@@ -0,0 +1,9 @@
+#ifndef HD_AAEON_H
+#define HD_AAEON_H
+
+#include "lcd.h"					  /* for Driver */
+
+// initialise this particular driver
+int hd_init_aaeon(Driver *drvthis);
+
+#endif
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/hd44780-advantech.c lcdproc-0.5.5/server/drivers/hd44780-advantech.c
--- lcdproc-0.5.5.orig/server/drivers/hd44780-advantech.c	1970-01-01 01:00:00.000000000 +0100
+++ lcdproc-0.5.5/server/drivers/hd44780-advantech.c	2012-09-05 15:42:08.725066309 +0200
@@ -0,0 +1,194 @@
+/** \file server/drivers/hd44780-advantech.c
+ * \c 8bit connection type of \c hd44780 driver for Hitachi HD44780 based LCD displays.
+ *
+ * The LCD is operated in its 8 bit-mode to be connected to a single PC parallel port.
+ */
+
+/* Copyright (c)  1999, 1995 Benjamin Tse <blt@Comports.com>
+ *                2001 Joris Robijn <joris@robijn.net>
+ *                
+ * Changed connection for Advantech FWA-3210 Network Appliance (internal LCM)
+ *                2012 Malte Pöggel <malte@maltepoeggel.de>
+ *
+ * The connections are:
+ * printer port   LCD
+ * D0   (2)  --- D0 (7) 
+ * D1   (3)  --- D1 (8)
+ * D2   (4)  --- D2 (9)
+ * D3   (5)  --- D3 (10)
+ * D4   (6)  --- D4 (11)
+ * D5   (7)  --- D5 (12)
+ * D6   (8)  --- D6 (13)
+ * D7   (9)  --- D7 (14)
+ * INIT (16) --- RS (4)
+ * nLF  (14) --- RW (5)
+ * nSTB (1)  --- EN (6)
+ *
+ * Backlight: nSEL (17)
+ *
+ * Keypad connection:
+ * Left  --- PE (12)
+ * Down  --- Busy (11)
+ * Up    --- nACK (10)
+ * Right --- Select (13)
+ * Enter --- nFAULT (15)
+ *
+ *
+ * Created modular driver Dec 1999, Benjamin Tse <blt@Comports.com>
+ *
+ * Based on the code in the lcdtime package which uses the LCD
+ * controller in its 8-bit mode.
+ *
+ * This file is released under the GNU General Public License. Refer to the
+ * COPYING file distributed with this package.
+ */
+
+#include "hd44780-advantech.h"
+#include "hd44780-low.h"
+#include "lpt-port.h"
+#include "port.h"
+#include "lcd_sem.h"
+#include "report.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+// Generally, any function that accesses the LCD control lines needs to be
+// implemented separately for each HW design. This is typically (but not
+// restricted to):
+// HD44780_senddata
+// HD44780_readkeypad
+
+void advantech_HD44780_senddata(PrivateData *p, unsigned char displayID, unsigned char flags, unsigned char ch);
+void advantech_HD44780_backlight(PrivateData *p, unsigned char state);
+unsigned char advantech_HD44780_scankeypad(PrivateData *p);
+
+#define RS	INIT
+#define RW	LF
+#define EN1	STRB
+#define BL      nSEL
+
+static int semid;
+
+
+/**
+ * Initialize the driver.
+ * \param drvthis  Pointer to driver structure.
+ * \retval 0       Success.
+ * \retval -1      Error.
+ */
+int
+hd_init_advantech(Driver *drvthis)
+{
+	PrivateData *p = (PrivateData*) drvthis->private_data;
+	HD44780_functions *hd44780_functions = p->hd44780_functions;
+
+	semid = sem_get();
+
+	// Reserve the port registers
+	if (port_access_multiple(p->port,3)) {
+		report(RPT_ERR, "%s: cannot get IO-permission for 0x%03X: %s",
+				drvthis->name, p->port, strerror(errno));
+		return -1;
+	}
+
+	hd44780_functions->senddata = advantech_HD44780_senddata;
+	hd44780_functions->backlight = advantech_HD44780_backlight;
+	hd44780_functions->scankeypad = advantech_HD44780_scankeypad;
+	
+	// setup the lcd in 8 bit mode
+	hd44780_functions->senddata(p, 0, RS_INSTR, FUNCSET | IF_8BIT);
+	hd44780_functions->uPause(p, 4100);
+	hd44780_functions->senddata(p, 0, RS_INSTR, FUNCSET | IF_8BIT);
+	hd44780_functions->uPause(p, 100);
+	hd44780_functions->senddata(p, 0, RS_INSTR, FUNCSET | IF_8BIT | TWOLINE | SMALLCHAR);
+	hd44780_functions->uPause(p, 40);
+
+	common_init (p, IF_8BIT);
+
+	return 0;
+}
+
+
+/**
+ * Send data or commands to the display.
+ * \param p          Pointer to driver's private data structure.
+ * \param displayID  ID of the display (or 0 for all) to send data to.
+ * \param flags      Defines whether to end a command or data.
+ * \param ch         The value to send.
+ */
+void
+advantech_HD44780_senddata(PrivateData *p, unsigned char displayID, unsigned char flags, unsigned char ch)
+{
+	unsigned char enableLines = 0, portControl;
+
+	// Only one controller is supported
+	enableLines = EN1;
+
+	if (flags == RS_INSTR)
+		portControl = 0;
+	else // RS_DATA
+		portControl = RS;
+
+	portControl |= p->backlight_bit;
+
+	sem_wait(semid);
+	port_out(p->port + 2, portControl ^ OUTMASK);
+	port_out(p->port, ch);
+	if (p->delayBus) p->hd44780_functions->uPause(p, 1);
+	port_out(p->port + 2, (enableLines|portControl) ^ OUTMASK);
+	if (p->delayBus) p->hd44780_functions->uPause(p, 1);
+	port_out(p->port + 2, portControl ^ OUTMASK);
+	sem_signal(semid);
+}
+
+
+/**
+ * Turn display backlight on or off.
+ * \param p      Pointer to driver's private data structure.
+ * \param state  New backlight status.
+ */
+void advantech_HD44780_backlight(PrivateData *p, unsigned char state)
+{
+	p->backlight_bit = (state ? BL : 0);
+	port_out(p->port + 2, p->backlight_bit ^ OUTMASK);
+}
+
+
+/**
+ * Read keypress.
+ * \param p      Pointer to driver's private data structure.
+ * \param YData  Bitmap of rows / lines to enable.
+ * \return       Bitmap of the pressed keys.
+ */
+unsigned char advantech_HD44780_scankeypad(PrivateData *p)
+{
+	unsigned char readval;
+
+	sem_wait(semid);
+
+	// Read inputs
+	readval = ~port_in(p->port + 1) ^ INMASK;
+
+	sem_signal(semid);
+
+        // Keys down?
+        if(readval & PAPEREND)          /* pin 15 */
+         {
+          return 1;
+         } else if(readval & BUSY)      /* pin 13 */
+          {
+           return 2;
+          } else if(readval & nACK)     /* pin 12 */
+           {
+            return 3;
+           } else if(readval & SELIN)   /* pin 11 */
+            {
+             return 4;
+            } else if(readval & nFAULT) /* pin 10 */
+             {
+              return 5;
+             }
+        return 0;
+}
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/hd44780-advantech.h lcdproc-0.5.5/server/drivers/hd44780-advantech.h
--- lcdproc-0.5.5.orig/server/drivers/hd44780-advantech.h	1970-01-01 01:00:00.000000000 +0100
+++ lcdproc-0.5.5/server/drivers/hd44780-advantech.h	2012-09-05 15:42:08.725066309 +0200
@@ -0,0 +1,9 @@
+#ifndef HD_ADVANTECH_H
+#define HD_ADVANTECH_H
+
+#include "lcd.h"					  /* for Driver */
+
+// initialise this particular driver
+int hd_init_advantech(Driver *drvthis);
+
+#endif
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/hd44780-drivers.h lcdproc-0.5.5/server/drivers/hd44780-drivers.h
--- lcdproc-0.5.5.orig/server/drivers/hd44780-drivers.h	2011-08-14 14:29:16.000000000 +0200
+++ lcdproc-0.5.5/server/drivers/hd44780-drivers.h	2012-09-05 15:42:22.170271744 +0200
@@ -16,6 +16,8 @@
 # include "hd44780-ext8bit.h"
 # include "hd44780-serialLpt.h"
 # include "hd44780-winamp.h"
+# include "hd44780-aaeon.h"
+# include "hd44780-advantech.h"
 #endif
 #include "hd44780-serial.h"
 #include "hd44780-lis2.h"
@@ -51,6 +53,8 @@
 	{ "8bit",          HD44780_CT_8BIT,          IF_TYPE_PARPORT, hd_init_ext8bit   },
 	{ "serialLpt",     HD44780_CT_SERIALLPT,     IF_TYPE_PARPORT, hd_init_serialLpt },
 	{ "winamp",        HD44780_CT_WINAMP,        IF_TYPE_PARPORT, hd_init_winamp    },
+	{ "aaeon",         HD44780_CT_AAEON,         IF_TYPE_PARPORT, hd_init_aaeon     },
+	{ "advantech",     HD44780_CT_ADVANTECH,     IF_TYPE_PARPORT, hd_init_advantech },
 #endif
 	/* serial connection types */
 	{ "picanlcd",      HD44780_CT_PICANLCD,      IF_TYPE_SERIAL,  hd_init_serial    },
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/hd44780-low.h lcdproc-0.5.5/server/drivers/hd44780-low.h
--- lcdproc-0.5.5.orig/server/drivers/hd44780-low.h	2011-08-14 14:29:16.000000000 +0200
+++ lcdproc-0.5.5/server/drivers/hd44780-low.h	2012-09-05 15:42:08.726066323 +0200
@@ -51,6 +51,8 @@
 #define HD44780_CT_USS720		18
 #define HD44780_CT_USBLCD		19
 #define HD44780_CT_USBTINY		20
+#define HD44780_CT_AAEON                21
+#define HD44780_CT_ADVANTECH            22
 
 // symbolic names for interface types
 #define IF_TYPE_UNKNOWN		0
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/Makefile.am lcdproc-0.5.5/server/drivers/Makefile.am
--- lcdproc-0.5.5.orig/server/drivers/Makefile.am	2011-08-14 14:29:16.000000000 +0200
+++ lcdproc-0.5.5/server/drivers/Makefile.am	2012-09-05 15:42:08.727066330 +0200
@@ -89,7 +89,7 @@
 glcdlib_SOURCES =    lcd.h lcd_lib.h glcdlib.h glcdlib.c report.h
 glk_SOURCES =        lcd.h glk.c glk.h glkproto.c glkproto.h report.h
 hd44780_SOURCES =    lcd.h lcd_lib.h hd44780.h hd44780.c hd44780-drivers.h hd44780-low.h hd44780-charmap.h report.h adv_bignum.h
-EXTRA_hd44780_SOURCES = hd44780-4bit.c hd44780-4bit.h hd44780-ext8bit.c hd44780-ext8bit.h lcd_sem.c lcd_sem.h hd44780-serialLpt.c hd44780-serialLpt.h hd44780-serial.c hd44780-serial.h hd44780-winamp.c hd44780-winamp.h hd44780-bwct-usb.c hd44780-bwct-usb.h hd44780-lcd2usb.c hd44780-lcd2usb.h hd44780-usbtiny.c hd44780-usbtiny.h hd44780-lis2.c hd44780-lis2.h hd44780-i2c.c hd44780-i2c.h hd44780-ftdi.c hd44780-ftdi.h hd44780-ethlcd.c hd44780-ethlcd.h hd44780-uss720.c hd44780-uss720.h hd44780-usblcd.c hd44780-usblcd.h port.h lpt-port.h timing.h
+EXTRA_hd44780_SOURCES = hd44780-4bit.c hd44780-4bit.h hd44780-ext8bit.c hd44780-ext8bit.h hd44780-aaeon.c hd44780-aaeon.h hd44780-advantech.c hd44780-advantech.h lcd_sem.c lcd_sem.h hd44780-serialLpt.c hd44780-serialLpt.h hd44780-serial.c hd44780-serial.h hd44780-winamp.c hd44780-winamp.h hd44780-bwct-usb.c hd44780-bwct-usb.h hd44780-lcd2usb.c hd44780-lcd2usb.h hd44780-usbtiny.c hd44780-usbtiny.h hd44780-lis2.c hd44780-lis2.h hd44780-i2c.c hd44780-i2c.h hd44780-ftdi.c hd44780-ftdi.h hd44780-ethlcd.c hd44780-ethlcd.h hd44780-uss720.c hd44780-uss720.h hd44780-usblcd.c hd44780-usblcd.h port.h lpt-port.h timing.h
 i2500vfd_SOURCES =   lcd.h i2500vfd.c i2500vfd.h glcd_font5x8.h report.h
 icp_a106_SOURCES =   lcd.h lcd_lib.h icp_a106.c icp_a106.h report.h
 imon_SOURCES =       lcd.h lcd_lib.h hd44780-charmap.h imon.h imon.c report.h adv_bignum.h
diff -NaurwB lcdproc-0.5.5.orig/server/drivers/Makefile.in lcdproc-0.5.5/server/drivers/Makefile.in
--- lcdproc-0.5.5.orig/server/drivers/Makefile.in	2011-11-09 20:56:34.000000000 +0100
+++ lcdproc-0.5.5/server/drivers/Makefile.in	2012-09-05 15:42:08.728066339 +0200
@@ -485,7 +485,7 @@
 glcdlib_SOURCES = lcd.h lcd_lib.h glcdlib.h glcdlib.c report.h
 glk_SOURCES = lcd.h glk.c glk.h glkproto.c glkproto.h report.h
 hd44780_SOURCES = lcd.h lcd_lib.h hd44780.h hd44780.c hd44780-drivers.h hd44780-low.h hd44780-charmap.h report.h adv_bignum.h
-EXTRA_hd44780_SOURCES = hd44780-4bit.c hd44780-4bit.h hd44780-ext8bit.c hd44780-ext8bit.h lcd_sem.c lcd_sem.h hd44780-serialLpt.c hd44780-serialLpt.h hd44780-serial.c hd44780-serial.h hd44780-winamp.c hd44780-winamp.h hd44780-bwct-usb.c hd44780-bwct-usb.h hd44780-lcd2usb.c hd44780-lcd2usb.h hd44780-usbtiny.c hd44780-usbtiny.h hd44780-lis2.c hd44780-lis2.h hd44780-i2c.c hd44780-i2c.h hd44780-ftdi.c hd44780-ftdi.h hd44780-ethlcd.c hd44780-ethlcd.h hd44780-uss720.c hd44780-uss720.h hd44780-usblcd.c hd44780-usblcd.h port.h lpt-port.h timing.h
+EXTRA_hd44780_SOURCES = hd44780-4bit.c hd44780-4bit.h hd44780-ext8bit.c hd44780-ext8bit.h hd44780-aaeon.c hd44780-aaeon.h hd44780-advantech.c hd44780-advantech.h lcd_sem.c lcd_sem.h hd44780-serialLpt.c hd44780-serialLpt.h hd44780-serial.c hd44780-serial.h hd44780-winamp.c hd44780-winamp.h hd44780-bwct-usb.c hd44780-bwct-usb.h hd44780-lcd2usb.c hd44780-lcd2usb.h hd44780-usbtiny.c hd44780-usbtiny.h hd44780-lis2.c hd44780-lis2.h hd44780-i2c.c hd44780-i2c.h hd44780-ftdi.c hd44780-ftdi.h hd44780-ethlcd.c hd44780-ethlcd.h hd44780-uss720.c hd44780-uss720.h hd44780-usblcd.c hd44780-usblcd.h port.h lpt-port.h timing.h
 i2500vfd_SOURCES = lcd.h i2500vfd.c i2500vfd.h glcd_font5x8.h report.h
 icp_a106_SOURCES = lcd.h lcd_lib.h icp_a106.c icp_a106.h report.h
 imon_SOURCES = lcd.h lcd_lib.h hd44780-charmap.h imon.h imon.c report.h adv_bignum.h
@@ -784,6 +784,8 @@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-bwct-usb.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-ethlcd.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-ext8bit.Po@am__quote@
+@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-aaeon.Po@am__quote@
+@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-advantech.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-ftdi.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-i2c.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hd44780-hd44780-lcd2usb.Po@am__quote@
@@ -929,6 +931,34 @@
 @AMDEP_TRUE@@am__fastdepCC_FALSE@	DEPDIR=$(DEPDIR) $(CCDEPMODE) $(depcomp) @AMDEPBACKSLASH@
 @am__fastdepCC_FALSE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -c -o hd44780-hd44780-ext8bit.obj `if test -f 'hd44780-ext8bit.c'; then $(CYGPATH_W) 'hd44780-ext8bit.c'; else $(CYGPATH_W) '$(srcdir)/hd44780-ext8bit.c'; fi`
 
+hd44780-hd44780-aaeon.o: hd44780-aaeon.c
+@am__fastdepCC_TRUE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -MT hd44780-hd44780-aaeon.o -MD -MP -MF $(DEPDIR)/hd44780-hd44780-aaeon.Tpo -c -o hd44780-hd44780-aaeon.o `test -f 'hd44780-aaeon.c' || echo '$(srcdir)/'`hd44780-aaeon.c
+@am__fastdepCC_TRUE@	$(am__mv) $(DEPDIR)/hd44780-hd44780-aaeon.Tpo $(DEPDIR)/hd44780-hd44780-aaeon.Po
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	source='hd44780-aaeon.c' object='hd44780-hd44780-aaeon.o' libtool=no @AMDEPBACKSLASH@
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	DEPDIR=$(DEPDIR) $(CCDEPMODE) $(depcomp) @AMDEPBACKSLASH@
+@am__fastdepCC_FALSE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -c -o hd44780-hd44780-aaeon.o `test -f 'hd44780-aaeon.c' || echo '$(srcdir)/'`hd44780-aaeon.c
+
+hd44780-hd44780-aaeon.obj: hd44780-aaeon.c
+@am__fastdepCC_TRUE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -MT hd44780-hd44780-aaeon.obj -MD -MP -MF $(DEPDIR)/hd44780-hd44780-aaeon.Tpo -c -o hd44780-hd44780-aaeon.obj `if test -f 'hd44780-aaeon.c'; then $(CYGPATH_W) 'hd44780-aaeon.c'; else $(CYGPATH_W) '$(srcdir)/hd44780-aaeon.c'; fi`
+@am__fastdepCC_TRUE@	$(am__mv) $(DEPDIR)/hd44780-hd44780-aaeon.Tpo $(DEPDIR)/hd44780-hd44780-aaeon.Po
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	source='hd44780-aaeon.c' object='hd44780-hd44780-aaeon.obj' libtool=no @AMDEPBACKSLASH@
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	DEPDIR=$(DEPDIR) $(CCDEPMODE) $(depcomp) @AMDEPBACKSLASH@
+@am__fastdepCC_FALSE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -c -o hd44780-hd44780-aaeon.obj `if test -f 'hd44780-aaeon.c'; then $(CYGPATH_W) 'hd44780-aaeon.c'; else $(CYGPATH_W) '$(srcdir)/hd44780-aaeon.c'; fi`
+
+hd44780-hd44780-advantech.o: hd44780-advantech.c
+@am__fastdepCC_TRUE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -MT hd44780-hd44780-advantech.o -MD -MP -MF $(DEPDIR)/hd44780-hd44780-advantech.Tpo -c -o hd44780-hd44780-advantech.o `test -f 'hd44780-advantech.c' || echo '$(srcdir)/'`hd44780-advantech.c
+@am__fastdepCC_TRUE@	$(am__mv) $(DEPDIR)/hd44780-hd44780-advantech.Tpo $(DEPDIR)/hd44780-hd44780-advantech.Po
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	source='hd44780-advantech.c' object='hd44780-hd44780-advantech.o' libtool=no @AMDEPBACKSLASH@
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	DEPDIR=$(DEPDIR) $(CCDEPMODE) $(depcomp) @AMDEPBACKSLASH@
+@am__fastdepCC_FALSE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -c -o hd44780-hd44780-advantech.o `test -f 'hd44780-advantech.c' || echo '$(srcdir)/'`hd44780-advantech.c
+
+hd44780-hd44780-advantech.obj: hd44780-advantech.c
+@am__fastdepCC_TRUE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -MT hd44780-hd44780-advantech.obj -MD -MP -MF $(DEPDIR)/hd44780-hd44780-advantech.Tpo -c -o hd44780-hd44780-advantech.obj `if test -f 'hd44780-advantech.c'; then $(CYGPATH_W) 'hd44780-advantech.c'; else $(CYGPATH_W) '$(srcdir)/hd44780-advantech.c'; fi`
+@am__fastdepCC_TRUE@	$(am__mv) $(DEPDIR)/hd44780-hd44780-advantech.Tpo $(DEPDIR)/hd44780-hd44780-advantech.Po
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	source='hd44780-advantech.c' object='hd44780-hd44780-advantech.obj' libtool=no @AMDEPBACKSLASH@
+@AMDEP_TRUE@@am__fastdepCC_FALSE@	DEPDIR=$(DEPDIR) $(CCDEPMODE) $(depcomp) @AMDEPBACKSLASH@
+@am__fastdepCC_FALSE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -c -o hd44780-hd44780-advantech.obj `if test -f 'hd44780-advantech.c'; then $(CYGPATH_W) 'hd44780-advantech.c'; else $(CYGPATH_W) '$(srcdir)/hd44780-advantech.c'; fi`
+
 hd44780-lcd_sem.o: lcd_sem.c
 @am__fastdepCC_TRUE@	$(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) $(hd44780_CFLAGS) $(CFLAGS) -MT hd44780-lcd_sem.o -MD -MP -MF $(DEPDIR)/hd44780-lcd_sem.Tpo -c -o hd44780-lcd_sem.o `test -f 'lcd_sem.c' || echo '$(srcdir)/'`lcd_sem.c
 @am__fastdepCC_TRUE@	$(am__mv) $(DEPDIR)/hd44780-lcd_sem.Tpo $(DEPDIR)/hd44780-lcd_sem.Po
